framework/source/class/qx/log/Logger.js

```diff
@@ -144,11 +155,26 @@ qx.Bootstrap.define("qx.log.Logger",
         return;
       }
 
-      delete this.__appender[id];
+      delete this.__appendersByName[appender.classname];
+      delete this.__appenders[id];
       delete appender.$$id;
     },
 
 
+    /**
+     * Adds a filter that specifies the appenders to use for a given logger name (classname)
+     * @param logger {String|RegExp} the pattern to match in the logger name
+     * @param appenderName {String?} the name of the appender class, if undefined then all appenders
+     * @param level {String?} the minimum logging level to use the appender, if undefined the default level is used
+     */
```

Comment:
having an example in the javadoc would be very helpful in understanding the intention of this.